eventuate-tram-cdc-mysql-connector/src/main/java/io/eventuate/tram/cdc/mysql/connector/configuration/ActiveMQMessageTableChangesToDestinationsConfiguration.java

```diff
@@ -17,7 +19,11 @@ public PublishingFilter activeMQDuplicatePublishingDetector() {
   }
 
   @Bean
-  public DataProducerFactory activeMQDataProducerFactory(@Value("${activemq.url}") String activeMQURL) {
-    return () -> new EventuateActiveMQProducer(activeMQURL);
+  public DataProducerFactory activeMQDataProducerFactory(@Value("${activemq.url}") String activeMQURL,
```

Comment:
@Value("${activemq.url}") seems to occur multiple times. why?

Reply:
I do not see duplicates, one line with it is removed and other line is added.

Reply:
Sorry. I was referring to the user id/password: `@Value("${activemq.user:#{null}}") `

Reply:
url, userid, user password used in several place.
I want to create a separate property class in the new projects (separate cdc).
Probably it should be done in common application.